src/components/TodoCreatingForm/TodoCreatingForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef } from 'react';
+import cn from 'classnames';
+import { useTodos } from '../TodosProvider';
+import { errorMessages } from '../ErrorNotification';
+
+export const TodoCreatingForm: React.FC = () => {
+  const {
+    newTodoTitle,
+    setNewTodoTitle,
+    createNewTodo,
+    setErrorMessage,
+    todos,
+    isLoading,
+  } = useTodos();
+
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (inputRef.current && !inputRef.current.disabled) {
+      inputRef.current.focus();
+    }
+  });
```

Comment:
It's bad practice to use useEffect without dependencies array but in some cases, it can be only the solution.

Try to double-check your code, maybe we can avoid it here